src/Compilers/CSharp/Test/Semantic/IOperation/IOperationTests_ILocalFunctionStatement.cs

```diff
@@ -458,5 +459,111 @@ void M(int p)
 
             VerifyOperationTreeAndDiagnosticsForTest<LocalFunctionStatementSyntax>(source, expectedOperationTree, expectedDiagnostics);
         }
+
+        [CompilerTrait(CompilerFeature.IOperation)]
+        [Fact, WorkItem(24650, "https://github.com/dotnet/roslyn/issues/24650")]
+        public void TestInvalidLocalFunction_ExpressionAndBlockBody()
+        {
+            string source = @"
+class C
+{
+    void M(int p)
+    {
+        /*<bind>*/object F() => new object(); { return null; }/*</bind>*/;
```

Comment:
The `{ return null; }` is part of the outer method, not the local function.

Reply:
Right. I wanted to document this in an explicit IOperation test though so that we can confirm the behavior from the parsing in IOperation.

---
In reply to: [178340655](https://github.com/dotnet/roslyn/pull/25782#discussion_r178340655) [](ancestors = 178340655)